vgen/src/test/java/io/gapi/vgen/testdata/java_settings_library.baseline

```diff
@@ -244,7 +253,14 @@ public class LibraryServiceSettings extends ServiceApiSettings {
    * Returns an instance of this class with recommended defaults.
    */
   public static LibraryServiceSettings defaultInstance() throws IOException {
-    return newBuilder().build();
+    return defaultBuilder().build();
+  }
+
+  /**
+   * Returns a builder for this class with recommended defaults.
+   */
+  public static Builder defaultBuilder() {
```

Comment:
If we have a defaultBuilder, I don't think we want a defaultInstance. I have been thinking about this and I have concluded that having a defaultBuilder is safer than defaultInstance because creating an instance actually creates a channel and an executor; having a build() step is less likely to be called indiscriminately (creating resources that won't be used) as compared to a defaultInstance() call, which implies something low-cost.